qcengine/programs/adcc.py

```diff
@@ -84,6 +84,16 @@ def compute(self, input_model: "AtomicInput", config: "TaskConfig") -> "AtomicRe
         model = input_model.model
         conv_tol = input_model.keywords.get("conv_tol", 1e-6)
 
+        need_one_of_states = {"n_spin_flip", "n_triplets", "n_states", "n_singlets"}
+        if not (need_one_of_states & set(input_model.keywords)):
+            input_model.keywords["n_states"] = 1
+
+        if input_model.driver not in ["energy", "properties"]:
+            raise InputError(f"Driver {input_model.driver} not implemented for ADCC.")
+
+        if not input_model.model.basis:
+            raise InputError("Method must contain a basis set.")
```

Comment:
@maxscheurer @mfherbst, I was adding ADCC to the basic checks over at test_harness_canonical.py and needed to make a few adjustments so that a molecule+model is sufficient. are these ok? should n_states be larger?

Reply:
Depends what you want ... I suppose if you go for excited states you rarely only want one. Also just asking for one can be numerically unreliable. I'd go for at least 3.

Reply:
thanks, will change it to 3.

I'm guessing the no-keywords case is targeting the user who's at the vague "I want the exited state" stage, probably meaning the lowest. So 3 should catch it.

Reply:
Yes, I suppose.

But IMHO the code should crash in this case, however, as at that level you're better off being forced to think a little what you actually want to compute before running the calculation. That's the policy we take in adcc. In particular in this default mode of operation there are no spin guarantees made ... so you'll probably end up finding a triplet state as lowest state, which might anyway not be what you are after.

Reply:
I'd suggest to set `n_singlets=3` to make the canonical tests work, but throw at least warning like
```Python
warnings.warn("No user-specified parameter for number of states for adcc. Computing 3 singlet states."
```
or something of the sort.

Reply:
Be careful: This will cause errors for unrestricted references, so we should have an extra branch for that case (where `n_states=3` is the only option).